torch/testing/_internal/common_utils.py

```diff
@@ -1981,11 +2021,12 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
             replace_with = torch.tensor(1, device=device, dtype=dtype)
         elif dtype in floating_types_and(torch.half, torch.bfloat16):
             replace_with = torch.tensor(torch.finfo(dtype).eps, device=device, dtype=dtype)
-        else:
-            assert dtype in complex_types()
+        elif dtype in complex_types():
             float_dtype = torch.float if dtype is torch.cfloat else torch.double
             float_eps = torch.tensor(torch.finfo(float_dtype).eps, device=device, dtype=float_dtype)
             replace_with = torch.complex(float_eps, float_eps)
+        else:
+            raise ValueError(f"Invalid dtype passed, supported dtypes are: {get_all_dtypes()}")
```

Comment:
I wouldn't make this change because we don't know what `get_all_dtypes()` will return in the future